### PR TITLE
ci: stop waiting for windows builds before deploying

### PR DESCRIPTION
**Pull request by blacha**
Windows builds are taking 5+ minutes linux build and deploy complete in around 5 minutes

### CODE REVIEW

File: .github/workflows/push.yml [hunk 1]
```diff
@@ -7,24 +7,15 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [macos-latest, windows-latest, ubuntu-latest]
+        os: [macos-latest, windows-latest]
     steps:
       - uses: linz/action-typescript@v1
 
-      - name: Check Dependencies
```
Comment (blacha) [inline .github/workflows/push.yml:14]: all of these checks were double run in `deploy`

File: .github/workflows/push.yml [hunk 2]
```diff
@@ -7,24 +7,15 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [macos-latest, windows-latest, ubuntu-latest]
```
Comment (Wentao-Kuang) [inline .github/workflows/push.yml:10]: Can we remove this ubuntu-latest, I see there is a expected `build (ubuntu-latest)` waiting in the ci.

Comment (blacha) [reply .github/workflows/push.yml:10]: Yup, just need to merge this

Comment (Wentao-Kuang) [reply .github/workflows/push.yml:10]: Okey, Thanks.

**Review by Wentao-Kuang — APPROVED** [review]

**Pull request merged**